test/validation/api/pool/pool.c

```diff
@@ -1698,7 +1698,19 @@ static void pool_ext_init_packet_pool_param(odp_pool_ext_param_t *param)
 static void test_packet_pool_ext_capa(void)
 {
 	odp_pool_ext_capability_t capa;
-	odp_pool_type_t type = ODP_POOL_PACKET;
+	odp_pool_type_t type;
+	odp_pool_type_t unsupported_types[4] = {ODP_POOL_BUFFER, ODP_POOL_TIMEOUT, ODP_POOL_VECTOR,
+						ODP_POOL_DMA_COMPL};
+	const int num_types = sizeof(unsupported_types) / sizeof(unsupported_types[0]);
```

Comment:
No need to be this PR, but an array size helper macro could also be useful (now that MIN/MAX ones are incoming #1886).

Reply:
True, I made a note to add a helper macro.